charts/iter8/templates/_experiment.tpl

```diff
@@ -0,0 +1,26 @@
+{{- define "experiment" -}}
```

Comment:
This enables the user to compose a new experiment type without having to create (or even understand about ) experiment charts.